packages/bpk-component-datatable/src/utils.js

```diff
@@ -21,3 +21,13 @@ export const hasClassName = (node, className) => {
   const nodeClassName = node.getAttribute('class');
   return nodeClassName && nodeClassName.split(' ').indexOf(className) !== -1;
 };
+
+export const remToPx = value => {
```

Comment:
I've copied this function from [bpk-component-barchart](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-barchart/src/utils.js#L29), not sure if this should be placed somewhere else and reused 😅